### DISCUSSION TITLE
fossify.org down?

### DISCUSSION THREAD

**Discussion by Enigma1309** [FossifyOrg/General-Discussion#31] [General]
The GoDaddy domain seems to be not working for now, please update it before someone else takes over that domain or/else use it as a malware attack vector 

![image](https://github.com/FossifyX/General-Discussion/assets/94917588/d1f98625-88ff-490f-95bf-53480d90aaec)

Recent Screenshot (6th December)

@naveensingh please consider looking forward to this issue

**Comment by naveensingh** [comment]
Both http://fossify.org and https://fossifyx.github.io seem to be working properly. Can you try clearing your cookies/site settings?

HTTPS is currently not enforced, I'm working on that.

**Reply by naveensingh** [reply]
![image](https://github.com/FossifyX/General-Discussion/assets/36371707/5d63c0bd-fcb5-41c7-abd0-9be251459a05)

**Reply by Enigma1309** [reply]
okay i got this, my browser is trying to upgrade http to https and this setting is turned on by default on most browsers.
Secondly you have set https link instead of http on the front page of fossify on github this leads to not loading the page for those users who clicked the link from the front page.
(image below)
![image](https://github.com/FossifyX/General-Discussion/assets/94917588/cffcecbf-aa36-4e89-a188-839b16828d17)
 
Enforcing HTTPS on site should seem to fix the issue. Thanks

**Comment by naveensingh** [comment]
Closing as HTTPS is now enforced.

**Reply by Enigma1309** [reply]
That's good 🙌
